### ISSUE TITLE
Filtering does not take effect when a collection has a tag in its URL

### ISSUE THREAD

**Issue by MegsParkinson** [Shopify/dawn#653]
**Describe the current behavior**

When a menu item is a collection with a product tag as a filter, the filter function does not filter the products.
![image](https://user-images.githubusercontent.com/82345267/133956301-0a1e4567-8988-4285-92c3-93d8b6a51b95.png)

[Screenshot removed]

**Describe the expected behavior**

The filter should filter the products in the collection, even though they have been already filtered by product tag in the navigation menu.

**Version information (Dawn, browsers and operating systems)**
- Dawn Version: 2.0.0 - 2.1.0
- Chrome Version 93.0.4577.82
- macOS Version 11.6

**Possible solution**
None found

**Additional context/screenshots**
Was reported previously, issue has been closed but not resolved: [#129](https://github.com/Shopify/dawn/issues/129)

**Comment by tairli** [comment]
I guess, #129 was closed because it's not a theme but backend problem.

**Comment by TGIFelix** [comment]
Having the same issue. Did anyone find a fix?

**Comment by rcobiella** [comment]
Same here. Any workarounds?

**Comment by MegsParkinson** [comment]
I suggested to my merchant that they create automated collections using the product tags and use those new collections in their menu. They've done this and it works, so for now I'm using that as a workaround.

**Comment by salvodees** [comment]
Hi @MegsParkinson, I have been investigating this issue and our devs confirmed the issue you are reporting is not exactly the same as this one: [#129](https://github.com/Shopify/dawn/issues/129).

Tag filtering is considered more of a hack and not a scalable/dynamic design choice like the new filtering system is, therefore not seeing the filters on a collection page where a tag has been used as a filter is an expected behaviour and the result of an explicit product decision.

In other words, a merchant can use _tags for filtering_ OR use _filters for filtering_. They can not be used together.

On the other hand, what wasn't expected is that the filters showed up in the theme with a tag applied. They should not have been showing up. That's what was fixed in [this issue](https://github.com/Shopify/dawn/issues/129).

I hope this clarifies and if there are no further questions please feel free to close this issue.
